save-cloud-common/src/commonMain/kotlin/org/cqfn/save/domain/FileInfo.kt

```diff
@@ -16,6 +16,24 @@ data class FileInfo(
     val uploadedMillis: Long,
     val sizeBytes: Long,
     val isExecutable: Boolean = false,
+) {
+    /**
+     * @return FileInfo dto
+     */
+    fun toDto(): FileInfoDto = FileInfoDto(
+        this.name,
+        this.isExecutable,
+    )
+}
+
+/**
+ * @property name name of a file
+ * @property isExecutable whether the file is executable
+ */
+@Serializable
+data class FileInfoDto(
```

Comment:
This name kind of contradicts usual convention Entity/Dto. Maybe, call this class `ShortFileInfo` or rename the original class into `VerboseFileInfo` or `FileInfoWithMetadata`?

Reply:
Renamed to `ShortFileInfo`